jetty-util/src/test/java/org/eclipse/jetty/util/resource/FileSystemResourceTest.java

```diff
@@ -658,7 +656,7 @@ public void testSymlink(Class<PathResource> resourceClass) throws Exception
         {
             // if unable to create symlink, no point testing the rest
             // this is the path that Microsoft Windows takes.
-            assumeTrue(true, "Not supported");
+            assumeTrue(false, "Not supported");
```

Comment:
This is an "assume" and it was written incorrectly, as `assumeTrue(true, "foo")` never does anything. It's the equiv of `assertTrue(true, "bar")`.

It might make more sense if you see the "long form" version of this same assumption ...

``` java
boolean symlinkSupported;
try
{
  Files.createSymbolicLink(bar, foo);
  symlinkSupported = true;
}
catch (UnsupportedOperationException | FileSystemException e)
{
  symlinkSupported = false;
}
assumeTrue(symlinkSupported); // break out of test if symlink isn't supported
doTestThatRequiresSymlinkSupport();
```